### PR TITLE
Roland DJ-505: Use let/const instead of var

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: res/controllers/Roland_DJ-505-scripts.js [hunk 1]
```diff
@@ -275,10 +273,11 @@ DJ505.browseEncoder = new components.Encoder({
     },
     input: function(channel, control, value, status, _group) {
         switch (status) {
-        case 0xBF: // Rotate.
-            var rotateValue = (value === 127) ? -1 : ((value === 1) ? 1 : 0);
+        case 0xBF: { // Rotate.
+            const rotateValue = (value === 127) ? -1 : ((value === 1) ? 1 : 0);
             this.onKnobEvent(rotateValue);
             break;
+        }
```
Comment (Swiftb0y) [inline res/controllers/Roland_DJ-505-scripts.js:280]: the extra braces are weird, I assume this is from eslint?

File: res/controllers/Roland_DJ-505-scripts.js [hunk 2]
```diff
@@ -1463,8 +1462,8 @@ DJ505.RollMode = function(deck, offset) {
     this.minSize = 0.03125;  // 1/32
     this.maxSize = 32;
 
-    var loopSize;
-    var i;
+    let loopSize;
+    let i;
     for (i = 0; i <= 3; i++) {
         loopSize = (this.loopSize * Math.pow(2, i));
```
Comment (Swiftb0y) [inline res/controllers/Roland_DJ-505-scripts.js:1468]: can't this both be moved into the loop? I don't see where it is used some other place in the function.
```suggestion
    for (let i = 0; i <= 3; i++) {
        const loopSize = (this.loopSize * Math.pow(2, i));
```

**Review by Swiftb0y — CHANGES_REQUESTED** [review]

File: res/controllers/Roland_DJ-505-scripts.js (hunk 1 above)
Comment (Holzhaus) [reply res/controllers/Roland_DJ-505-scripts.js:280]: Without the extra braces, it causes an eslint error. Similar to c++, where declaring a variable inside a switch case without braces is also not allowed.

Comment (Swiftb0y) [reply res/controllers/Roland_DJ-505-scripts.js:280]: right, makes sense. I just didn't know the extra braces were valid syntax.

**Review by Swiftb0y — APPROVED** [review]
Thanks LGTM

**Pull request merged**